components/clientComponents/forms/FormattedDate/FormattedDate.tsx

```diff
@@ -147,12 +147,15 @@ export const FormattedDate = (props: FormattedDateProps): React.ReactElement =>
               <label className="mb-2" htmlFor={`${name}-${part}`}>
                 {t(`formattedDate.${part}`)}
               </label>
```

Comment:
The aria-described by with the "number" description may be overkill. I've never seen this done before on an input but it may help to identify the input as accepting numbers.  We could try with user testing.